### PR TITLE
doc fixes

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
 [![Docs](https://docs.rs/tokio-bin-process/badge.svg)](https://docs.rs/tokio-bin-process)
 [![dependency status](https://deps.rs/repo/github/shotover/tokio-bin-process/status.svg)](https://deps.rs/repo/github/shotover/tokio-bin-process)
 
-Allows your integration tests to run your application under a separate process and assert on tokio-tracing events.
+Allows your integration tests to run your application under a separate process and assert on `tracing` events.
```
Comment (rukai) [inline readme.md:7]: hmmm, maybe `tokio-rs/tracing` and/or link to https://github.com/tokio-rs/tracing
Even though the project is called tracing, just `tracing` is ungoogleable and ambiguous.

File: readme.md [hunk 2]
```diff
@@ -72,19 +72,19 @@ async fn test_some_functionality() {
...
 ## Application side setup
 
 There is a non-trival amount of setup required in the application itself.
 You will need to:
 
-* Provide a way to set tracing to output in json mode
-* Handle panics as a `tracing::error!` in json mode.
-* Ensure SIGTERM cleanly shutsdown the application and exits with a return code
+* Provide a way to set tracing to output in JSON mode.
+b* Handle panics as a `tracing::error!` in JSON mode.
```
Comment (rukai) [inline readme.md:87]: `b`

File: tokio-bin-process/src/event.rs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-//! Structures to represent an event created by tokio tracing
+//! Structures to represent an event created by `tracing`.
```
Comment (rukai) [inline tokio-bin-process/src/event.rs:1]: same as my earlier comment on the name

**Review by rukai — APPROVED** [review]
Nice!
Thanks for the cleanup

**Review by benbromhead — APPROVED** [review]

**Pull request merged**